client/src/components/applicants/Applicants.js

```diff
@@ -30,7 +30,7 @@ export class Applicants extends Component {
     const { fetchApplicants, fetchJobPosting, adminView } = this.props
     const postingId = window.location.href.split('/')[4]
     if (nProps.loggedIn && !this.state.isLoaded && !adminView) {
-      fetchJobPosting(postingId)
+      fetchJobPosting(postingId, nProps.loggedIn)
```

Comment:
The second argument of `fetchJobPosting` is never used in the action, apparently left in the code by accident, so not needed here, either